components/stepper/index.css

```diff
@@ -12,378 +12,439 @@
-				--mod-stepper-focus-indicator-visibility: visible;
-			}
-		}
+	&:not(.is-disabled).is-keyboardFocused,
+	&:not(.is-disabled):focus-visible,
+	&:not(.is-disabled):focus-within {
+		--mod-numberfield-border-color: var(--highcontrast-numberfield-border-color-focus, var(--mod-numberfield-border-color-keyboard-focus, var(--spectrum-numberfield-border-color-keyboard-focus)));
 
-		&.is-invalid {
-			--mod-stepper-border-color: var(--mod-stepper-border-color-invalid, var(--spectrum-negative-border-color-default));
+		&.is-hover,
+		&:hover {
+			--mod-numberfield-border-color: var(--highcontrast-numberfield-border-color-focus-hover, var(--mod-numberfield-border-color-focus-hover, var(--spectrum-numberfield-border-color-focus-hover)));
 		}
 	}
-}
-
-/* --- High contrast settings --- */
-@media (forced-colors: active) {
-	.spectrum-Stepper {
-		--highcontrast-stepper-border-color: CanvasText;
-		--highcontrast-stepper-border-color-hover: Highlight;
-		--highcontrast-stepper-border-color-focus: Highlight;
-		--highcontrast-stepper-border-color-focus-hover: Highlight;
-		--highcontrast-stepper-border-color-keyboard-focus: CanvasText;
-		--highcontrast-stepper-focus-indicator-color: Highlight;
 
-		&.is-invalid {
-			--highcontrast-stepper-border-color: Highlight;
-			--highcontrast-stepper-border-color-hover: Highlight;
-			--highcontrast-stepper-border-color-focus: Highlight;
-			--highcontrast-stepper-border-color-focus-hover: Highlight;
-			--highcontrast-stepper-border-color-keyboard-focus: Highlight;
-		}
-
-		&.is-disabled {
-			--highcontrast-stepper-border-color: GrayText;
-			--highcontrast-stepper-buttons-border-width: var(--mod-stepper-border-width, var(--spectrum-stepper-border-width));
-		}
-
-		&:not(.is-disabled) {
-			&:hover {
-				--highcontrast-stepper-border-color: var(--highcontrast-stepper-border-color-hover);
-			}
+	&.is-disabled {
+		--mod-numberfield-border-color: var(--spectrum-numberfield-border-color-disabled);
 
-			&.is-focused,
-			&:focus {
-				--highcontrast-stepper-border-color: var(--highcontrast-stepper-border-color-focus);
+		/* --mod-numberfield-background-color: var(--spectrum-numberfield-background-color-disabled); */
+		--mod-numberfield-background-color: var(--spectrum-gray-25);
+	}
 
-				&:hover {
-					--highcontrast-stepper-border-color: var(--highcontrast-stepper-border-color-focus-hover);
-				}
-			}
+	&:lang(ja),
+	&:lang(zh),
+	&:lang(ko) {
+		--spectrum-numberfield-line-height: var(--spectrum-cjk-line-height-100);
+	}
 
-			&.is-keyboardFocused,
-			&:focus-visible {
-				--highcontrast-stepper-border-color: var(--highcontrast-stepper-border-color-keyboard-focus);
-			}
-		}
+	/* @passthrough start -- MODS for sub components */
+	/* nested textfield */
+	--mod-textfield-corner-radius: var(--mod-numberfield-border-radius, var(--spectrum-numberfield-border-radius));
+	--mod-textfield-border-width: var(--mod-numberfield-border-width, var(--spectrum-numberfield-border-width));
```

Comment:
I have another comment below because I saw this with inline-size first, but it looks like we're doing passthroughs to textfield that are passed through but being overridden for border width and some other properties:
![image](https://github.com/user-attachments/assets/6c519edb-c616-4060-9d99-493b03824b27)

Looks like the border, for instance, is also being set on `.spectrum-NumberField-textfield`:
![image](https://github.com/user-attachments/assets/aaadd34c-a159-4f01-b0e6-d57ef02a0554)